### ISSUE TITLE
Error migrating from a Typed Link Field nested inside Vizy field

### ISSUE THREAD

**Issue by d-karstens** [verbb/hyper#62]
### Describe the bug

Hi,

We've been running into an error migrating a typed link field that's inside a Vizy field

The error message shown is

```
Preparing to migrate field “button” (1a3e388e-2ac8-4b75-ae79-6be5536728d6).
Exception: Calling unknown method: verbb\hyper\migrations\MigrateTypedLinkField::convertModel() (/var/www/html/vendor/yiisoft/yii2/base/Component.php:300)
#0 /var/www/html/vendor/verbb/hyper/src/migrations/PluginMigration.php(147): yii\base\Component->__call('convertModel', Array)
#1 /var/www/html/vendor/verbb/vizy/src/services/Content.php(235): verbb\hyper\migrations\PluginMigration->verbb\hyper\migrations\{closure}('button', Array)
#2 /var/www/html/vendor/verbb/hyper/src/migrations/PluginMigration.php(154): verbb\vizy\services\Content->modifyFieldContent('1a3e388e-2ac8-4...', 'button', Object(Closure), Object(craft\db\Connection))
#3 /var/www/html/vendor/verbb/hyper/src/migrations/MigrateTypedLinkField.php(177): verbb\hyper\migrations\PluginMigration->migrateVizyContent(Array)
#4 /var/www/html/vendor/verbb/hyper/src/migrations/PluginFieldMigration.php(38): verbb\hyper\migrations\MigrateTypedLinkField->processFieldSettings()
#5 /var/www/html/vendor/craftcms/cms/src/db/Migration.php(49): verbb\hyper\migrations\PluginFieldMigration->safeUp()
#6 /var/www/html/vendor/verbb/hyper/src/controllers/MigrationsController.php(68): craft\db\Migration->up()
#7 /var/www/html/vendor/verbb/hyper/src/controllers/MigrationsController.php(34): verbb\hyper\controllers\MigrationsController->_migrate('verbb\\hyper\\mig...')
#8 [internal function]: verbb\hyper\controllers\MigrationsController->actionTypedLinkField()
#9 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#10 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#11 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('typed-link-fiel...', Array)
#12 /var/www/html/vendor/craftcms/cms/src/web/Application.php(302): yii\base\Module->runAction('hyper/migration...', Array)
#13 /var/www/html/vendor/craftcms/cms/src/web/Application.php(627): craft\web\Application->runAction('hyper/migration...', Array)
#14 /var/www/html/vendor/craftcms/cms/src/web/Application.php(281): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#15 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#16 /var/www/html/web/index.php(13): yii\base\Application->run()
#17 {main}
```

The field type gets migrated properly, but content is missing

Let me know if you need any more info,
Thanks!

### Steps to reproduce

1. Have a typed link field nested inside a Vizy block
2. Run `craft hyper/migrate/typed-link-field` and see error

### Craft CMS version

4.4.10.1

### Plugin version

1.1.5

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed in [1.1.6](https://github.com/verbb/hyper/releases/tag/1.1.6)

**Comment by d-karstens** [comment]
Cheers!
